web-components/src/components/modal/Modal.ts

```diff
@@ -271,8 +270,7 @@ export namespace Modal {
                   role="dialog"
                   id="${this.htmlId}"
                   class="md-modal ${classMap(this.modalContainerClassMap)}"
-                  aria-label="${this.ariaLabel}"
-                  aria-labelledby=${this.ariaLabelledBy}
```

Comment:
Why are we removing aria-labelledby here, as per the recommendation the dialog title needs to be associated using aria-labelledby.

Reply:
if we add aria-labelledby it is reading everything thats in header , which includes the close button.
we need it to read only the title and exclude the close button @neelabhnagaich

Reply:
most modals are already sending headerLabel, reset and station login which currently does not send headerLabe are not impacted by this change.

Reply:
I guess we should keep the labelled by as it is. Did you check are we using this property in our desktop we don't want that functionality to be broken because of this.